package/chainConfigs.ts

```diff
@@ -0,0 +1,22 @@
+export const chainConfigs = {
+  [1]: {
```

Comment:
is this auto-generated?

if not, would make sense to use constant names for chain ids?

Reply:
It's generated automatically by this script:
https://github.com/ourzora/zora-1155-contracts/pull/93/files#diff-d58480676bf837f4a98dacf641cfd02442c21ad0555ba3a53a8cda64cf8f5350

Constant names would be great - but it's just right now generating this from whats in chainConfigs folder